pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -380,9 +383,9 @@ def test_optimize(
 
     # create a new repo version, sync again, assert not optimized
     repository = rpm_repository_api.read(repository.pulp_href)
-    content = choice(get_content(repository.to_dict())[RPM_PACKAGE_CONTENT_NAME])
```

Comment:
@dralley, I am unsure if it is the correct approach to use randomization techniques in tests. The tests should be reproducible, and we do not store the seed.

Reply:
Actually, there is no guarantee that Pulp returns items in the same order. So, there is still some randomization incorporated.